MdeModulePkg/Universal/Console/TerminalDxe/TerminalConIn.c

```diff
@@ -78,6 +78,9 @@ TerminalConInReset (
     );
 
   Status = TerminalDevice->SerialIo->Reset (TerminalDevice->SerialIo);
+  // MU_CHANGE [BEGIN] - Ensure serial device is ready to receive data
+  Status = TerminalDevice->SerialIo->SetControl (TerminalDevice->SerialIo, EFI_SERIAL_DATA_TERMINAL_READY|EFI_SERIAL_REQUEST_TO_SEND);

```

Comment:
The status check below previously applied the `Reset()` call but it's now lost and `Status` refers to the `SetControl()` return value. It would be nice to still handle the return status from `Reset()`.

Reply:
Good point, the report status code is changed.

Reworked code to set control bits after the report status code is checked, and only if serialio reset was successful.